### PR TITLE
config: add an option to disable chunk

### PR DESCRIPTION
**Pull request by coocood**
In case we need to use the old row executor.

### CODE REVIEW

**Comment by coocood** [comment]
@shenli PTAL

File: config/config.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ type Config struct {
 	RunDDL       bool   `toml:"run-ddl" json:"run-ddl"`
 	SplitTable   bool   `toml:"split-table" json:"split-table"`
 	TokenLimit   int    `toml:"token-limit" json:"token-limit"`
+	DisableChunk bool   `toml:"disable-chunk" json:"disable-chunk"`
```
Comment (ngaut) [inline config/config.go]: s/DisableChunk/EnableChunk

Comment (shenli) [reply config/config.go]: DisableChunk is better. If we do not set it, it should be false.

File: server/conn.go [hunk 2]
```diff
@@ -790,7 +790,7 @@ func (cc *clientConn) handleFieldList(sql string) (err error) {
 // resultsets, it's used to support the MULTI_RESULTS capability in mysql protocol.
 func (cc *clientConn) writeResultset(goCtx goctx.Context, rs ResultSet, binary bool, more bool) error {
 	defer terror.Call(rs.Close)
-	if rs.SupportChunk() {
+	if !cc.server.cfg.DisableChunk && rs.SupportChunk() {
```
Comment (shenli) [inline server/conn.go]: Could we check  DisableChunk  in SupportChunk?

**Comment by coocood** [comment]
@ngaut 
PTAL

File: server/conn.go (hunk 2 above)
Comment (coocood) [reply server/conn.go]: The ResultSet don't have the config to check.

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**